scripts/api.js

```diff
@@ -0,0 +1,116 @@
+import PocketChange from './pocket-change.js';
+import MacroSupport from './macro-support.js';
+import Settings from './settings.js';
+
+const API = {
+	PocketChange: undefined,
+    MacroSupport: undefined,
+
+    /**
+    * For all selected tokens, generate currency for them
+	*
+	* @param {boolean} - (optional) if true it will generate a random currency without the rating check
+    */
+	generateCurrencyForSelectedTokens(ignoreRating = false) {
+		if(!this.MacroSupport){
```

Comment:
Are these guards even possible? It doesn't seem like they would be and could probably be removed considering everything should be ready on the `setup` hook

Reply:
removed